### ISSUE TITLE
Liquid error with tag {% seo %}

### ISSUE THREAD

**Issue by lucamodica** [PandaSekh/Jekyll-Podcaster#8]
While I was trying to compile my Jekyll site with the theme, I encountered the following error: 

```
Liquid syntax error (PATH_TO_RUBY/Ruby30-x64/lib/ruby/gems/3.0.0/gems/jekyll-podcaster-1.0.0/_includes/head.html line 33): Unknown tag 'seo' included  (Liquid::SyntaxError)
```

**Comment by PandaSekh** [comment]
It seems like you are missing the jekyll-seo-tag dependency.
Can you please try to install all the deps with `bundle install`?

Thanks

**Comment by lucamodica** [comment]
I added `gem "jekyll-podcaster"` to the Gemfile file followed by the `bundle install` command, but still trying to compile it I couldn't solve the issue.

**Comment by lucamodica** [comment]
I added `gem "jekyll-seo-tag"` to the Gemfile file followed by the `bundle install` command, but still trying to compile it I couldn't solve the issue.

**Comment by PandaSekh** [comment]
I tried to create a new Jekyll project following the [official docs](https://jekyllrb.com/docs/) and then I installed this theme following my own instructions for the [gem-based theme](https://github.com/PandaSekh/Jekyll-Podcaster#gem-based-theme) (I don't know if that's what you followed but it seems like it).

Indeed the project broke as you said. I checked online and it seems like a strange issue with Jekyll, Ruby and Bundler, but I was able to fix it.

In the `_config.yml` file, add the jekyll-seo-tag plugin as a dependency:
```yml
plugins:
  - jekyll-seo-tag
```

And also add the gem in the Gemfile:
```yml
gem "jekyll-seo-tag"
```

Delete the `Gemfile.lock`, run `bundle install` and retry.
